src/Compilers/CSharp/Portable/Syntax/NullableDirectiveMap.cs

```diff
@@ -45,9 +57,11 @@ private NullableDirectiveMap(ImmutableArray<(int Position, bool? State)> directi
         /// `enable` or `safeonly`, false if `disable`, and null if no preceding directive,
```

Comment:
> or `safeonly` [](start = 20, length = 14)

Looks like I missed some mentions of `safeonly`. Can you update this comment, as well as the similar comment on `CSharpSyntaxTree.GetNullableDirectiveState`

Reply:
Done